pyat/at/physics/orbit.py

```diff
@@ -76,7 +76,7 @@ def find_orbit4(ring, dp=0.0, refpts=None, guess=None, **kwargs):
     delta_matrix = numpy.zeros((6, 5), order='F')
     for i in range(4):
         delta_matrix[i, i] = step_size
-    id4 = numpy.asfortranarray(numpy.identity(4))
+    id4 = numpy.asfortranarray(numpy.eye(4))
```

Comment:
I'm not sure why you've changed `identity` to `eye` here.

Reply:
Because it was causing a problem with our threading in atip and identity just imports and calls eye anyway. So I figured there was no downside to changing it.

Reply:
Except it was renamed to `identity` because it is a clearer name. I vote to reinstate it because it is easier to understand.

Reply:
I have no problem with `eye` since it's the name for the similar Matlab function, but I let you, python experts decide.